lms/djangoapps/instructor_task/tasks.py

```diff
@@ -191,6 +193,7 @@ def get_course_forums_usage(entry_id, xmodule_instance_args):
     return run_main_task(entry_id, task_fn, action_name)
 
 
+# pylint: disable=not-callable
```

Comment:
Why do we need to disable pylint for this existing method?  Is it raising linter exceptions?